support/package_nginx

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+#
+# This script is used for the [Nginx
+# buildpack](github.com/Scalingo/nginx-buildpack/). Please refer to this page
+# (https://www.notion.so/scalingooriginal/New-Nginx-Version-936b92e173a74f5ab0cd59e5155a4880)
+# on how to use it.
+#
+
+set -e
+
+basedir="$( cd -P "$( dirname "$0" )" && pwd )"
+source "$basedir/lib/utils"
+source "$basedir/lib/s3"
+source "$basedir/../config/versions.sh"
+
+# Binding between modsecurity and nginx (since modsecurity can be used as standalone or with apache for instance)
+MODSECURITY_NGINX_VERSION="v1.0.2"
+MODSECURITY_NGINX_DIR="modsecurity-nginx-${MODSECURITY_NGINX_VERSION}"
+MODSECURITY_NGINX_TARBALL="${MODSECURITY_NGINX_DIR}.tar.gz"
+MODSECURITY_NGINX_URL="https://github.com/SpiderLabs/ModSecurity-nginx/releases/download/${MODSECURITY_NGINX_VERSION}/${MODSECURITY_NGINX_TARBALL}"
+
+nginx_version="${1:-$default_nginx_version}"
+zlib_version="$default_zlib_version"
+
+validate_env
+
+export PATH=${basedir}/../vendor/bin:$PATH
+
+if [ -z "$nginx_version" ]; then
+  echo "Usage: $0 <version>" >&2
+  exit $E_ARG_MISSING
+fi
+
+if [ -z "$NGINX_PCRE_VERSION" ]; then
+  # From https://sourceforge.net/projects/pcre/files/pcre/
+  NGINX_PCRE_VERSION=8.45
```

Comment:
Should we make this configurable via the env?

Reply:
It is configurable actually, there is a condition for that.

But being frank we never changed it